Completion/Unix/Command/_find

```diff
@@ -110,6 +112,7 @@ case $variant in
       '*-readable'
       '*-writable'
       '*-xtype:file type:((b\:block\ special\ file c\:character\ special\ file d\:directory p\:named\ pipe f\:normal\ file l\:symbolic\ link s\:socket))'
+      '-files0-from[search NUL separated paths from FILE]:file:_path'
```

Comment:
The description can't say FILE since the list of options doesn't say `-files0-from FILE`.

`_path` doesn't exist.  Should be `_files`, probably.

How about:

```suggestion
      '-files0-from[start recursing from targets in given file]:NUL-separated targets file:_files'
```